### PR TITLE
Updates to changelog

### PR DESCRIPTION
**Pull request by jasongrout**
## User-facing changes

This adds more polish to the changelog.

### CODE REVIEW

**Review by afshin — APPROVED** [review]
:+1:

File: docs/source/getting_started/changelog.rst [hunk 1]
```diff
@@ -13,67 +13,66 @@ milestone on GitHub for the full list of pull requests and issues closed.
...
-
-In JupyterLab 3.0, extensions are typically pip or conda packages that are dynamically loaded by the application (no more nodejs build step needed). See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
+In JupyterLab 3.0, a new recommended way of distributing and installing extensions as Python pip or conda packages is available. Installing such extensions does not require rebuilding JupyterLab and does not require having NodeJS installed. The previous way of distributing extensions as npm packages requiring rebuilding JupyterLab is still available as well. See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
 
 The JupyterLab interface is now translatable
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 JupyterLab now provides the ability to set the display language of the user interface.
-See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/language.html>`__ for more details.
+See the `internationalization documentation <https://jupyterlab.readthedocs.io/en/latest/user/language.html>`__ for more details.
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:24]:
```suggestion
See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/language.html>`__ for more details.
```

File: docs/source/getting_started/changelog.rst [hunk 2]
```diff
@@ -13,67 +13,66 @@ milestone on GitHub for the full list of pull requests and issues closed.
...
-~~~~~~~~~~~~~~~~~~~~~~~
-JupyterLab now ships with a Debugger front-end by default.  See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/debugger.html>`__ for more details.
+A new visual debugger
+~~~~~~~~~~~~~~~~~~~~~
+
+JupyterLab now ships with a debugger front-end by default, available for kernels that support the new debugging protocol.  See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/debugger.html>`__ for more details.
 
 
 Improvements to Single Document Mode and Mobile 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (afshin) [inline docs/source/getting_started/changelog.rst:35]: The single-document user interface has been improved to be more streamlined. Both modes of JupyterLab now support showing the current file in use in the browser URL bar, similar to the classic notebook UI.

File: docs/source/getting_started/changelog.rst [hunk 3]
```diff
@@ -13,67 +13,66 @@ milestone on GitHub for the full list of pull requests and issues closed.
...
 
-
-Extensions are now Python packages and do not require ``node``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In JupyterLab 3.0, extensions are typically pip or conda packages that are dynamically loaded by the application (no more nodejs build step needed). See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
+In JupyterLab 3.0, a new recommended way of distributing and installing extensions as Python pip or conda packages is available. Installing such extensions does not require rebuilding JupyterLab and does not require having NodeJS installed. The previous way of distributing extensions as npm packages requiring rebuilding JupyterLab is still available as well. See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
 
 The JupyterLab interface is now translatable
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:22]:
```suggestion
The JupyterLab interface supports multiple languages
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Pull request merged**